network/stations.csv

```diff
@@ -593,6 +593,7 @@ AICS,SM,Aickens,-42.768185088,171.624357133,268,,WGS84,2003-08-18T00:00:00Z,2008
 AIRO,XX,Airedale,-45.00131,170.91063,262,,WGS84,2014-09-18T00:00:00Z,2015-04-15T02:00:00Z
 AKCZ,SI,Akaroa Harbour,-43.8721,172.90983,374.2,,WGS84,2012-03-01T00:00:00Z,9999-01-01T00:00:00Z
 AKFZ,RA,Atkinson Farm,-36.605669,174.522439,94,,WGS84,2023-03-21T02:06:00Z,9999-01-01T00:00:00Z
+AKLS,SM,Auckland University,-36.853647,174.769832,69.756,,WGA84,2023-06-23T01:07:00Z,9999-01-01T00:00:00Z
```

Comment:
Same issue there  - 70  meters is good when verified 
Provided that fromer AKUS is 50 meters

Reply:
@salichon coordinates updated